hello_bot/hello_bot.py

```diff
@@ -58,10 +52,7 @@ def teamswebhook():
 
 
 if __name__ == '__main__':
-
-    # Read the configuration that contains the bot access token
-    config = read_yaml_data(os.getcwd() + '\config\config.yaml')['hello_bot']
-    teams_api = WebexTeamsAPI(access_token=config['teams_access_token'])
+    teams_api = WebexTeamsAPI(access_token='')
```

Comment:
We can discuss later if the access token should be a global variable at the beginning of this file or we should add instructions to add it here

Reply:
It depends on what happens if you make multiple WebexTeamsAPI objects with the same token. Does it still work?